docs/kh2/file/type/msn.md

```diff
@@ -4,34 +4,29 @@ Mission file. They are located in `msn/{language}/` and describe how a certain m
 
 ## Mission
 
-This is the entry point of the file. It is a binary-type file and it always have as a name `{WORLD_ID}{MAP_INDEX}`
+This is the entry point of the file. It is a binary-type file and it's name is always `{WORLD_ID}{MAP_INDEX}`. Example: for the file `EH21_MS101`, where `EH` is the [world](../../worlds.md) and `21` is the map index, the mission name is called `EH21`.
```

Comment:
Potential typo: `the mission name is called **MS101**`

Reply:
I guess i wrote that a bit unclear. What i meant is the first subfile of an MSN file always follows the {world}{room} naming scheme.
![grafik](https://user-images.githubusercontent.com/29046361/102325993-2130a080-3f84-11eb-85bb-901749926747.png)